src/renderer/components/blocks/tabs/MyCommittedUnitsTab.tsx

```diff
@@ -0,0 +1,73 @@
+import { FormattedMessage } from 'react-intl';
+import { ProjectModal, SkeletonTable, UnitsListTable } from '@/components';
+import React, { useCallback, useEffect } from 'react';
+import { useGetUnitsQuery } from '@/api';
+import { useColumnOrderHandler, useQueryParamState, useWildCardUrlHash } from '@/hooks';
+import { debounce } from 'lodash';
+
+interface PageTabProps {
+  orgUid: string;
+  search: string;
+  setIsLoading: (isLoading: boolean) => void;
+}
+
+const MyCommittedUnitsTab: React.FC<PageTabProps> = ({ orgUid, search, setIsLoading }: PageTabProps) => {
+  const [currentPage, setCurrentPage] = useQueryParamState('page', '1');
+  const [order, setOrder] = useQueryParamState('order', undefined);
+  const handleSetOrder = useColumnOrderHandler(order, setOrder);
+  //todo: rename this fragment to reflect units and replace the projects modal with the units modal !!!!
```

Comment:
did you do this TODO?

Reply:
yes. I need to pull in the units modal from refactor base, but i wanted to commit prior to doing so. I dont like merging with uncommitted changes.

This todo will not exist when merged

Reply:
change of plans, there will be todo's becuase the units modal still needs to be added